src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -1,941 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We would need datetime transformer for forecasting scenarios

Reply:
Maybe we can move this transformer under Microsoft.ML.Transforms?

Reply:
It's not currently in a shippable state.  It depends on an unsupported native library.  So if we need this component it needs to be re-implemented.

Reply:
> It's not currently in a shippable state. It depends on an unsupported native library. So if we need this component it needs to be re-implemented.

I'm okay doing what makes the most sense here so long as the final outcome is the `FeaturizeDateTime` transform is exposed to users in a future release.

Reply:
> Maybe we can move this transformer under Microsoft.ML.Transforms?

Not in the current form.  @michaelgsharp mentioned offline that we could reimplement it completely in C#.  Let's make sure we have an issue tracking this request.  It can reference this PR for historical purposes in casee any of the code removed here is useful in the future.

Reply:
I reactivated https://github.com/dotnet/machinelearning/issues/6232 to track bringing this back in a different form.